ophyd/pseudopos.py

```diff
@@ -327,45 +361,54 @@ def target(self):
         '''Last commanded target positions'''
         return self.PseudoPosition(*(pos.target for pos in self._pseudo))
 
+    def _sequential_move(self, real_pos, timeout=None, **kwargs):
+        '''Move all real positioners to a certain position, in series'''
+        for real, value in zip(self._real, real_pos):
+            logger.debug('[sequential] Moving %s to %s (timeout=%s)',
+                         real.name, value, timeout)
+            if timeout <= 0:
+                raise TimeoutError('Failed to move all positioners within '
+                                   '%s s'.format(timeout))
+
+            t0 = time.time()
+            try:
+                real.move(value, wait=True, timeout=timeout, **kwargs)
```

Comment:
This means that we can not use sequential mode from bluesky (as we it needs non-blocking).  I suspect we need to build a callback chain here (each one finishing triggers the next one to start and the last one flips the status object.